### PR TITLE
feat: [PL-25735]: opa secrets response view

### PR DESCRIPTION
**Pull request by ujjawal-harness**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/34661)

### CODE REVIEW

File: 120-ng-manager/src/main/java/io/harness/ng/opa/OpaEvaluationContext.java [hunk 1]
```diff
@@ -12,14 +12,13 @@
...
 import lombok.Data;
+import lombok.experimental.SuperBuilder;
 
 @Data
-@Builder
+@SuperBuilder
 @OwnedBy(HarnessTeam.PL)
 public class OpaEvaluationContext {
-  Object entity;
   UserOpaEvaluationContext user;
```
Comment (scottyw) [inline 120-ng-manager/src/main/java/io/harness/ng/opa/OpaEvaluationContext.java:22]: We just agreed on slack that these user/action/date fields would be removed globally, which I think makes sense.

File: 120-ng-manager/src/main/java/io/harness/ng/opa/entities/connector/ConnectorOpaEvaluationContext.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import io.harness.annotations.dev.OwnedBy;
+import io.harness.ng.opa.OpaEvaluationContext;
+import lombok.Data;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder
+@OwnedBy(HarnessTeam.PL)
+public class ConnectorOpaEvaluationContext extends OpaEvaluationContext {
+    Object entity;
```
Comment (scottyw) [inline 120-ng-manager/src/main/java/io/harness/ng/opa/entities/connector/ConnectorOpaEvaluationContext.java:13]: This is the field name we will look at migrating to "connector" at a later time. 👍

**Review by scottyw — APPROVED** [review]

File: 120-ng-manager/src/main/java/io/harness/ng/opa/OpaEvaluationContext.java (hunk 1 above)
Comment (ujjawal-harness) [reply 120-ng-manager/src/main/java/io/harness/ng/opa/OpaEvaluationContext.java:22]: yes right @scottyw

File: 120-ng-manager/src/main/java/io/harness/ng/opa/entities/connector/ConnectorOpaEvaluationContext.java (hunk 2 above)
Comment (ujjawal-harness) [reply 120-ng-manager/src/main/java/io/harness/ng/opa/entities/connector/ConnectorOpaEvaluationContext.java:13]: right @scottyw

**Review by scottyw — APPROVED** [review]
> Reviewed 8 of 8 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/34661)* status: 8 file(s) to review. 2. unresolved discussions. 1 of 2 approvals obtained.

**Review by pbarapatre10 — APPROVED** [review]
> Reviewed 8 of 8 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/34661)* status: 2. unresolved discussions.

**Review by scottyw — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/34661)* status: 2. unresolved discussions.

**Review by scottyw — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/34661)* status: :shipit: complete! ✓

**Pull request merged**